### ISSUE TITLE
Source map not getting picked up in Google Chrome Developer due to missing sourceMappingURL

### ISSUE THREAD

**Issue by mrdungx** [yeoman/generator-webapp_DEPRECATED#461]
The source map file for CSS is generated correctly to .tmp/styles/main.css.map, however Google Chrome Developer does not pick it up.

Suggested fix: add this line to the end of main.scss:

``` css
/*# sourceMappingURL=main.css.map */
```

**Comment by aegyed91** [comment]
I had the exact same problem. It is because of grunt-autoprefixer, you need to specify an extra option in your Gruntfile. ( map: true )

`autoprefixer: {
            options: {
                map: true,
                browsers: ['> 1%', 'last 2 versions', 'Firefox ESR', 'Opera 12.1']
            },`

You can read more about it here: https://www.npmjs.org/package/grunt-autoprefixer#options-map

**Comment by eddiemonge** [comment]
PR welcome (with tests)

**Comment by alessandro-fazzi** [comment]
As far as I know the fix is already in master; isn't it?

``` js
        map: {
          prev: '.tmp/styles/'
        }
```

anyway I see that `yo install` will install the last tagged commit

**Comment by everthis** [comment]
same here.

**Comment by silvenon** [comment]
This has been fixed, try the new v0.6.0 :wink:
